ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/openssl3/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+##################################################################
+# Install OpenSSL 3.0.10.1000                                    #
+# An OpenSSL version greater than 1.1.x is needed for DNF        #
+# See https://github.com/adoptium/infrastructure/issues/3274
+##################################################################
+---
+- name: Check if openssl is installed
+  stat:
+    path: /usr/bin/openssl
+  register: openssl_installed
+
+- name: Get version of installed openssl (if installed else skip)
+  shell: /usr/bin/openssl version | awk '{print$2}' | awk -F. '{print$1}'
+  register: openssl_version
+  when: openssl_installed.stat.exists
+
+- name: Install openssl if not installed or version is less than 3
```

Comment:
Perhaps add comment with a link to the IBM site where people can download this from themselves.

Reply:
Done